app/views/compute_resources_vms/index/_ovirt.json.erb

```diff
@@ -0,0 +1,15 @@
+{ "sEcho": <%= params['sEcho'] %>,
```

Comment:
This tells datatables which request this is for (in case there are several concurrent ones)